boost/teamcity_boost.cpp

```diff
@@ -71,7 +94,7 @@ struct TeamcityFormatterRegistrar {
     }
 };
 
-BOOST_GLOBAL_FIXTURE(TeamcityFormatterRegistrar)
+BOOST_GLOBAL_FIXTURE(TeamcityFormatterRegistrar);
 
 // Formatter implementation
 static std::string toString(boost::unit_test::const_string bstr) {
```

Comment:
why not to move this to anonymous namespace instead of C style `static` keyword...